Woof/WoofTests/ThirdParty/NetworkService/NetworkServiceIntegrationTests.swift

```diff
@@ -3,6 +3,6 @@ import XCTest
 
 final class NetworkServiceIntegrationTests: XCTestCase {
     func testMethodTestUseFromNetworkServiceCanBeCalled() {
-        NetworkService().testUse()
+        _ = NetworkService<WoofAppEndpoint>.init()
```

Comment:
Why do we need to call init explicitly?
```suggestion
        _ = NetworkService<WoofAppEndpoint>()
```

Reply:
Fixed.